src/driver.cpp

```diff
@@ -1056,12 +1057,14 @@ void driver::build_fock_operator(const json &json_fock, Molecule &mol, FockBuild
         auto eps_i = json_fock["reaction_operator"]["epsilon_in"];
         auto eps_o = json_fock["reaction_operator"]["epsilon_out"];
         auto formulation = json_fock["reaction_operator"]["formulation"];
-        auto accelerate_pot = (optimizer == "potential") ? true : false;
+        auto accelerate_pot = (optimizer == "potential") ? true : false; // this has to go
```

Comment:
no need for the ternary `if`, but I guess that this whole statement is useless, given the comment? If so, just remove it.
```suggestion
        auto accelerate_pot = (optimizer == "potential"); // this has to go
```

Reply:
Yes, will get to fix the input file